### PR TITLE
Allow quarkus.container-image.group to be empty

### LINKED ISSUE #8117: quarkus.container-image.group could be blank
it's not mandatory to have a group for docker image in the repository for private docker repository.

so we have

quarkus.container-image.registry=docker.example.com:5000
quarkus.container-image.group=
quarkus.container-image.tag=latest

### PR DESCRIPTION
**Pull request by geoand**
Fixes: #8117

### CODE REVIEW

**Review by Ladicek — APPROVED** [review]

**Review by iocanel — APPROVED** [review]
LGTM

**Review by gastaldi — APPROVED** [review]

File: extensions/container-image/spi/src/main/java/io/quarkus/container/spi/ContainerImageInfoBuildItem.java [hunk 1]
```diff
@@ -51,18 +32,6 @@ public ContainerImageInfoBuildItem(Optional<String> registry, String group, Stri
...
-        return group;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public String getTag() {
-        return tag;
-    }
```
Comment (Ladicek) [inline extensions/container-image/spi/src/main/java/io/quarkus/container/spi/ContainerImageInfoBuildItem.java]: I've just found that removing these methods is actually SPI-breaking :-/ It was a bad SPI arguably, but still. Something to bear in mind for the future.

(And, well, I'll dig out the `name` out of the `image` :-) )

Comment (geoand) [reply extensions/container-image/spi/src/main/java/io/quarkus/container/spi/ContainerImageInfoBuildItem.java]: :+1: 

Better break now that no one is using it :P

Comment (Ladicek) [reply extensions/container-image/spi/src/main/java/io/quarkus/container/spi/ContainerImageInfoBuildItem.java]: Thanks for being so considerate :stuck_out_tongue_closed_eyes:

Comment (geoand) [reply extensions/container-image/spi/src/main/java/io/quarkus/container/spi/ContainerImageInfoBuildItem.java]: :laughing:

**Pull request merged**